<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoxOxCallBarFrame</class>
 <widget class="VoxOxFrame" name="VoxOxCallBarFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>30</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>30</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CallBar - VoxOx</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox{
	background: black;
	border: black;
}

QComboBox::drop-down:editable {
   background: black;
	border: black;
 }
 
 QComboBox::drop-down:editable :on{
    background: black;
	border: black;
 }</string>
  </property>
  <layout class="QGridLayout" name="callbarFrameLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>5</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="flagFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>19</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1000</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select Country</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="VoxOxClickableLabel" name="flagLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>19</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>19</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="VoxOxClickableLabel" name="areaCodeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="VoxOxClickableLabel" name="flagDropLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>12</width>
          <height>19</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>12</width>
          <height>19</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../../../../resources/wengophone.qrc">:/pics/callbar/arrow_down_white.png</pixmap>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="callBarComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1000</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
     <property name="duplicatesEnabled">
      <bool>true</bool>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VoxOxFrame</class>
   <extends>QFrame</extends>
   <header>qtutil/VoxOxFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VoxOxClickableLabel</class>
   <extends>QLabel</extends>
   <header>qtutil/VoxOxClickableLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../resources/wengophone.qrc"/>
  <include location="../../../../../resources/wengophone.qrc"/>
  <include location="../../../../../resources/wengophone.qrc"/>
  <include location="../../../../../resources/wengophone.qrc"/>
  <include location="../../../../../resources/wengophone.qrc"/>
  <include location="../../../../../resources/wengophone.qrc"/>
  <include location="../../../../../resources/wengophone.qrc"/>
 </resources>
 <connections/>
</ui>
